### ISSUE TITLE
Map is hard to see in the phone

### ISSUE THREAD

**Issue by jorgeguzmanecon** [infowetrust/startup-nation#22]
Not sure if this is an enhancement, or a must-fix, but the map is very hard to show in a regular phone, and it often happens that it is precisely when only a phone is available when it is the best time to show the map (i.e., people asking about details during dinners etc, rather than sitting on a computer).  More generally, one would expect tablets and mobiles to be at the core of how people choose to consume our platform.

Is there anything that can be done?

**Comment by infowetrust** [comment]
What if:
- Medium-term: you show it on a tablet in these instances. I will open a new issue to check for touch-pinch interaction for this use case.

- Longer-term: we may develop a smartphone skin. But we should approach this after Kauffman event.

**Comment by jorgeguzmanecon** [comment]
I don't think we have a a table to show etc.. but keeping this as post-kauffman for now would work fine.  I will look into getting  a tablet on my side (which should be fine, i think)
